dist/typings/proteus.d.ts

```diff
@@ -266,7 +266,7 @@ export declare module session {
   }
 
   abstract class PreKeyStore {
-    public prekeys: Array<number>;
+    public prekeys: Array<keys.PreKey>;
```

Comment:
Exactly, but the type of `prekeys` is up to the implementation. To support developers we could define the following types:

```typescript
public prekeys: Array<keys.PreKey> | Array<number>;
```

Do you agree with that?